presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraType.java

```diff
@@ -453,7 +487,7 @@ public Object validatePartitionKey(Object value)
             case MAP:
             default:
                 // todo should we just skip partition pruning instead of throwing an exception?
-                throw new PrestoException(NOT_SUPPORTED, "Unsupport partition key type: " + this);
+                throw new PrestoException(NOT_SUPPORTED, "Unsupported partition key type: " + this);
```

Comment:
It would be better to pack cleanup changes like this one or formatting fixes in a separate commit.